package.json

```diff
@@ -20,7 +20,8 @@
     "concurrently": "^5.3.0",
     "cross-env": "^7.0.2",
     "cross-var": "^1.1.0",
-    "dotenv-cli": "^3.2.0"
+    "dotenv-cli": "^3.2.0",
+    "react-icons": "^4.12.0"
```

Comment:
I'd prefer if we kept our icon usage in MUI's icon library if possible so we don't have to take on other dependencies.

Alternate icon suggestion:
https://mui.com/material-ui/material-icons/?query=warn&theme=Rounded

Reply:
@Spiteless  fixed